src/index.html

```diff
@@ -5,6 +5,21 @@
     <title>ALMighty</title>
     <meta charset="UTF-8">
     <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport">
+    <!-- Hack for search component css. Refactoring Needed-->
+    <style>
```

Comment:
@nimishamukherjee css is required before the load of the component, that's why i have placed it here. Need to find a solution.